src/commands/runDraftTool/runDraftDockerfile.ts

```diff
@@ -109,8 +109,14 @@ class PromptSourceCodeFolder extends AzureWizardPromptStep<WizardContext> {
 class PromptLanguage extends AzureWizardPromptStep<WizardContext> {
    public async prompt(wizardContext: WizardContext): Promise<void> {
       const languageToItem = (lang: DraftLanguage) => ({label: lang.name});
+      let infoLanguages: DraftLanguage[];
+      try {
+         infoLanguages = await getDraftLanguages();
+      }catch (error) {
+         throw Error('Failed to get Draft languages');
+      }
       const languageOptions: vscode.QuickPickItem[] =
-         draftLanguages.map(languageToItem);
+         infoLanguages.map(languageToItem);
       const languagePick = await wizardContext.ui.showQuickPick(
          languageOptions,
```

Comment:
We want to pass a promise into the `quickPick`. This will show a loading bar while it waits. In the current form it will look like nothing it happening while it runs the draft command which is confusing to the user.

Refactor the current thing slightly to get it to pass a promise to showQuickPick. Don't await getDraftLanguages. See how we show the quick pick for subscriptions as an example.